README.md

```diff
@@ -1,5 +1,6 @@
 <img alt="PHP" src="https://img.shields.io/badge/php-%23777BB4.svg?&style=for-the-badge&logo=php&logoColor=white"/> <img alt="JavaScript" src="https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E"/> <img alt="Python" src="https://img.shields.io/badge/python%20-%2314354C.svg?&style=for-the-badge&logo=python&logoColor=white"/> <img alt="CSS3" src="https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white"/>
-# Kidney Exchange Program <img alt="Brand Logo " src="https://github.com/siv2r/kidney-exchange/blob/master/images/logo3.png?raw=true" width="100" height="40">
+
+<h1><img  align="right;" alt="Brand Logo " src="https://github.com/siv2r/kidney-exchange/blob/master/images/logo3.png?raw=true" width="40" height="40">Kidney Exchange Program</h1>
```

Comment:
Could you decrease the width and height size so that it becomes equal to the heading